cranelift/interpreter/src/interpreter.rs

```diff
@@ -240,6 +240,29 @@ impl<'a> InterpreterState<'a> {
             _ => &self.frame_stack[num_frames - 1],
         }
     }
+
+    fn is_address_within_stack_slot_range(&self, addr: &Address, ty: &Type) -> bool {
+        let size = ty.bytes() as u64;
+        let addr_start = addr.offset;
+        let addr_end = addr_start + size;
+        let stack_slots = &self.get_current_function().sized_stack_slots;
```

Comment:
I saw that [`InterpreterState::stack_address`](https://github.com/bytecodealliance/wasmtime/blob/3ff3994a128e900dfe7a465c7f36fc5a074d07ef/cranelift/interpreter/src/interpreter.rs#L296) utilises `sized_stack_slots` and as far as I can tell the stack slot order is preserved by `PrimaryMap`.

However, I am not sure how to account for padding between the slots at this point in time.

Reply:
I see. Thanks for the clarification!